guides/09_other-tutorials/streaming-ai-generated-audio.md

```diff
@@ -0,0 +1,151 @@
+# Streaming AI Generated Audio
+
+Tags: AUDIO, STREAMING
+
+In this guide, we'll build a novel AI application to showcase Gradio's audio output streaming. We're going to a build a talking [Magic 8 Ball](https://en.wikipedia.org/wiki/Magic_8_Ball) 🎱
+
+A Magic 8 Ball is a toy that answers any question after you shake it. Our application will do the same but it will also speak its response!
+
+We won't cover all the implementation details in this blog post but the code is freely available on [Hugging Face Spaces](https://huggingface.co/spaces/gradio/magic-8-ball).
+
+## The Overview
+
+Just like the classic Magic 8 Ball, a user should ask it a question orally and then wait for a response. Under the hood, we'll use Whisper to transcribe the audio and then use an LLM to generate a magic-8-ball-style answer. Finally, we'll use Parler TTS to read the response aloud.
+
+## The UI
```

Comment:
Would be really cool if we could get the UI to "shake" with some custom JS to really mimic an 8 ball

Reply:
Yes I actually thought about this but then ran into this issue! https://github.com/gradio-app/gradio/pull/9188